index.js

```diff
@@ -35,6 +36,8 @@ function Feedsme(opts) {
     throw new Error('Feedsme URL required');
   }
 
+  this.version = (opts.version === '1' || opts.version === 'v1' || opts.version === 1) ? '' : 'v2';
```

Comment:
Same comment here as with the carpenterd client. I'm okay with defaulting to v2, but it should be a major semver bump.